integration_tests/test_household_structure.py

```diff
@@ -54,3 +52,53 @@ def test_household_id_and_address_id_correspond(tracked_live_populations):
     assert (all_time_pop.groupby("address_id")["household_id"].nunique() == 1).all()
     # Note, however, that the reverse is not true: a household_id can span multiple address_ids
     # (over multiple time steps) when the whole house moved as a unit between those time steps.
+
+
+def test_new_reference_person_is_oldest_household_member(tracked_live_populations):
+    for time_step in range(max(TIME_STEPS_TO_TEST)):
+        before_reference_person_idx = (
+            tracked_live_populations[time_step]
+            .loc[
+                tracked_live_populations[time_step]["relation_to_household_head"]
+                == "Reference person"
+            ]
+            .index
+        )
+        after_reference_person_idx = (
+            tracked_live_populations[time_step + 1]
+            .loc[
+                tracked_live_populations[time_step + 1]["relation_to_household_head"]
+                == "Reference person"
+            ]
+            .index
+        )
+        new_reference_person_idx = np.setdiff1d(
+            after_reference_person_idx, before_reference_person_idx
+        )
+
+        # Get households with new reference persons
+        household_ids_with_new_reference_person = tracked_live_populations[time_step + 1].loc[
+            new_reference_person_idx, "household_id"
+        ]
+        assert len(household_ids_with_new_reference_person) == len(
```

Comment:
This could only fail if there was a household with multiple reference people in it, right? I think we have a test somewhere else that tests that (if we don't we should and it should be its own test).